### DISCUSSION TITLE
Confused about how choose CLIP...

### DISCUSSION THREAD

**Discussion by ZeroCool22** [mcmonkeyprojects/SwarmUI#246] [Q&A] [answered]
There are 2 folders with the name CLIP on the backend...

I put the custom CLIP on the two folder to test, but on the GUI i can't choose the CLIP if i choose MODEL, i mean i can choose one or another, why is it?

![Screenshot_1](https://github.com/user-attachments/assets/c4f1023a-5014-4a62-b31a-e9536bcf73d9)

![Screenshot_6](https://github.com/user-attachments/assets/02669ce1-05ce-49f0-ac67-864525e8a345)

![Screenshot_2](https://github.com/user-attachments/assets/bac872e8-ec29-431b-9b87-6904cd46380e)

Would like to know how i can keep the CLIP and model at the same time selected.

**Comment by ZeroCool22** [comment, marked as answer]
Found it.

![Screenshot_3](https://github.com/user-attachments/assets/eb5cfc64-ba72-4b87-b172-ff825eafa447)

**Comment by mcmonkey4eva** [comment]
that `diffusion_models/clip` folder is one you made yourself by accident

Both of those paths are wrong regardless - You want `(Swarm)/Models/clip`
